sphinx_autobuild/build.py

```diff
@@ -28,9 +29,12 @@ def __call__(self, *, rebuild: bool = True):
             )
             raise
 
-        show(command=["sphinx-build"] + self.sphinx_args)
+        show(command=[sys.executable, "-m", "sphinx"] + self.sphinx_args)
```

Comment:
This might be nicer than `sys.executable` (avoiding the full path)?
```suggestion
        show(command=["python", "-m", "sphinx"] + self.sphinx_args)
```

Reply:
we'd have the same problem this avoids if we did that though.

Reply:
oh, just for show, yeah, sure, two secs

Reply:
fixed :)